projects/packages/identity-crisis/src/class-ui.php

```diff
@@ -161,4 +165,27 @@ function ( $c1, $c2 ) {
 		return static::$consumers;
 	}
 
+	/**
+	 * Get the site platform.
+	 *
+	 * @return string
+	 */
+	private static function get_platform() {
+		$host = new Host();
+
+		if ( $host->is_woa_site() ) {
```

Comment:
Should we also check for the Atomic platform in general? WoA is a subset of Atomic, and there are other environments that run on it such as Pressable... which IIRC has its own constant too.

Reply:
I believe Atomic is too broad, as we're trying to distinguish between the actual site types.
For example, if a Newspack site is hosted on Atomic we're more interested in knowing that it's a Newspack site because of their own toolset in creating new sites.

For Atomic we could add another flag (e.g. `is_atomic: true`), but it doesn't seem we would need it for this particular use case.